### PR TITLE
chore: add 'concepCy' to spacy universe

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description
This PR aims at adding [concepCy](https://github.com/JulesBelveze/concepcy) to the SpaCy universe.

### Types of change
Website changes

## Checklist
- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/meta/universe.json [hunk 1]
```diff
@@ -2604,7 +2638,7 @@
                 "    Add the courgette, garlic, red peppers and oregano and cook for 2–3 minutes.",
                 "    Later, add some oranges and chickens.\"\"\"",
                 "",
-                "# use any model that has internal spacy embeddings", 
+                "# use any model that has internal spacy embeddings",
```
Comment (polm) [inline website/meta/universe.json:2641]: I notice you got rid of unnecessary spaces in a couple of places. That's OK but out of curiosity, is there a particular preprocessor you used that did this?

Comment (JulesBelveze) [reply website/meta/universe.json:2641]: I didn't not notice it my bad. I'm using the default PyCharm code formatter so I guess must have been that

Comment (polm) [reply website/meta/universe.json:2641]: No worries, it's fine to include those changes, just wondered what caught it.

**Review by polm — CHANGES_REQUESTED** [review]
Thanks! This mostly looks good and I confirmed the code works. Could you fill in the `slogan` though? That's used on the main overview page of the Universe, which is why it's different from the `description`. The `description` can be pretty long if you want but the `slogan` should just be a sentence or two. You can look at other entries for examples.

**Review by polm — APPROVED** [review]
Looks good, thanks!

**Pull request merged**